### PR TITLE
Add sqlalchemy test and gcc for bcrypt and cryptography tests

### PR DESCRIPTION
**Pull request by odidev**
Added sqlalchemy test and gcc installation as it is required for building cffi for bcrypt and cryptography tests.

### CODE REVIEW

File: .github/workflows/centos8-wheel-tester.yaml [hunk 1]
```diff
@@ -53,9 +53,11 @@ jobs:
               from cryptography.hazmat.backends.openssl.backend import backend
               print('Loaded: ' + backend.openssl_version_text())
               print('Linked Against: ' + backend._ffi.string(backend._lib.OPENSSL_VERSION_TEXT).decode('ascii'))
+          - PIP_NAME: sqlalchemy
+            PKG_TEST: import sqlalchemy
     name: "Centos8 for ${{ matrix.WHEEL.PIP_NAME }}"
     steps:
-      - run: yum install -y python3-pip
+      - run: yum install -y python3-pip gcc
```
Comment (geoffreyblake) [inline .github/workflows/centos8-wheel-tester.yaml:60]: People trying to use python packages via pip should ultimately not have to have a development environment installed to be able to use Arm64 packages.  Therefore I'm placing the bar at: has to work only with a simple `pip install` same as x86.

File: .github/workflows/centos8-wheel-tester.yaml [hunk 2]
```diff
@@ -53,9 +53,11 @@ jobs:
               from cryptography.hazmat.backends.openssl.backend import backend
               print('Loaded: ' + backend.openssl_version_text())
               print('Linked Against: ' + backend._ffi.string(backend._lib.OPENSSL_VERSION_TEXT).decode('ascii'))
+          - PIP_NAME: sqlalchemy
```
Comment (geoffreyblake) [inline .github/workflows/centos8-wheel-tester.yaml:56]: Can you place this in the AL2 and Ubuntu testers as well? Thanks.

File: .github/workflows/centos8-wheel-tester.yaml (hunk 1 above)
Comment (odidev) [reply .github/workflows/centos8-wheel-tester.yaml]: @geoffreyblake I agree with you, removed development environment.

File: .github/workflows/centos8-wheel-tester.yaml (hunk 2 above)
Comment (odidev) [reply .github/workflows/centos8-wheel-tester.yaml:56]: @geoffreyblake I have added the test for AL2 and Ubuntu testers as well. Please review the changes.

**Pull request merged**